inc/connector/namespace.php

```diff
@@ -184,9 +236,10 @@ function pre_reschedule_event( $pre, $event ) {
  * @param int       $timestamp Timestamp for when to run the event.
```

Comment:
GitHub won't let me comment on the correct line, but the `$pre` parameter can also be a `WP_Error`: https://developer.wordpress.org/reference/hooks/pre_unschedule_event/

Reply:
Are you referring to updating the docblock here?

Reply:
Yes!

Reply:
Thought so! That's all done then :)